test/card/info-status/info-status-card.component.spec.js

```diff
@@ -73,7 +73,7 @@ describe('Component: pfInfoStatusCard', function () {
 
       element = compileCard('<pf-info-status-card status="status"></pf-info-status-card>', $scope)
```

Comment:
Missing semicolon, looks like in quite a few spots within the file...

Reply:
bahahahah eeeekkkk other project I'm on uses standard.js too funny, rubymine doesn't seem to respect that 🤦‍♂️